internal/rules/rbac/allowing_create_role_binding_and_associate_privileged_clusterrole_test.rego

```diff
@@ -0,0 +1,313 @@
+package builtin.kubernetes.KSV051
+
+test_allowing_create_role_binding_and_associate_clusterrole {
+	r := deny with input as {
+		"apiVersion": "rbac.authorization.k8s.io/v1",
+		"kind": "Role",
+		"metadata": {
+			"namespace": "default",
+			"name": "pod-reader",
+		},
+		"rules": [
+			{
+				"apiGroups": ["rbac.authorization.k8s.io"],
+				"resources": ["rolebindings"],
+				"verbs": ["create"],
+			},
+			{
+				"apiGroups": ["rbac.authorization.k8s.io"],
+				"resources": ["clusterroles"],
+				"verbs": ["bind"],
+				"resourceNames": ["*"],
+			},
+		],
+	}
+
+	count(r) > 0
```

Comment:
I think we should use `count(r) == 1` in these tests, as we want to make sure duplicates aren't reported to the user.